oml/functional/metrics.py

```diff
@@ -545,20 +527,20 @@ def calc_pcf(embeddings: Tensor, pfc_variance: Tuple[float, ...]) -> List[Tensor
 
 
 def extract_pos_neg_dists(
-    distances: Tensor, mask_gt: Tensor, mask_to_ignore: Optional[Tensor]
+    distances: Tensor,
+    mask_gt: Tensor,
 ) -> Tuple[Tensor, Tensor]:
     """
     Extract distances between relevant samples, and distances between non-relevant samples.
 
     Args:
         distances: Distance matrix with the shape of ``[query_size, gallery_size]``
         mask_gt: ``(i,j)`` element indicates if for i-th query j-th gallery is the correct prediction
-        mask_to_ignore: Binary matrix to indicate that some elements in gallery cannot be used
-                     as answers and must be ignored
     Returns:
         pos_dist: Tensor of distances between relevant samples
         neg_dist: Tensor of distances between non-relevant samples
     """
+    mask_to_ignore = torch.isinf(distances)
     if mask_to_ignore is not None:
```

Comment:
Don't need to check it anymore

Reply:
I want to have this check here in the case someone decides to call the function directly

Reply:
![Screenshot from 2023-03-05 10-56-51](https://user-images.githubusercontent.com/43877180/222953639-3200ecc3-c8bc-4b79-b5cd-0b08e6dc3d0d.png)
I mean now `mask_to_ignore` is `not None`, it's always `Tensor` (line 543).

Reply:
discussed offline